docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: "3.7"
+
+services:
+  db:
+    image: postgres:14
+    volumes:
+      - ./volumes/db:/var/lib/postgresql/data
+    env_file:
+      - .dockerenv
+      - .env.local_password
+  web:
+    build:
+      context: .
+      dockerfile: docker/apiserver/Dockerfile-local
```

Comment:
Would it be useful to name this file `docker-compose-local.yml`, for symmetry?

Reply:
this is just for dev env simplicity, by default `docker-compose up` assumes the file to be named docker-compose.yml And this mode will mostly be used by humans. If we do rename the file, `docker-compose -f docker-compose-local.yml up` will be needed, a bit verbose and non-intuitive.